### ISSUE TITLE
PostCSS 7 conflicts

### ISSUE THREAD

**Issue by redaxmedia** [postcss/postcss-custom-media#38]
Hello,

are you sure that `"postcss": "^7.0.2"` is absolutely needed? It causes other PostCSS plugins to fail that still need PostCSS 6 - therefore I had to downgrade your plugin.

Best regards
Henry

**Comment by ai** [comment]
PostCSS 7 plugins will work with PostCSS 6 runtime and other PostCSS 6 other plugins.

PostCSS 7 had no AST changes.

**Comment by redaxmedia** [comment]
Well, I got this error message - maybe this is not even related to the `postcss-custom-media` plugin then and just some side effect as it on of the last plugins upgraded to PostCSS 7!?

Working PostCSS setup:

```
		"postcss": "7.0.2",
		"postcss-color-function": "4.0.1",
		"postcss-color-gray": "4.1.0",
		"postcss-custom-media": "6.0.0",
		"postcss-custom-properties": "7.0.0",
		"postcss-custom-selectors": "5.1.0",
		"postcss-extend": "1.0.5",
		"postcss-import": "12.0.0",
		"postcss-nesting": "6.0.0",
		"postcss-reporter": "6.0.0",
		"postcss-rtl": "1.3.2",
		"stylefmt": "6.0.2",
```

Upgrading the postcss-custom-media to `7.0.1` plugin is causing the error:

```
Unknown error from PostCSS plugin. Your current PostCSS version is 6.0.23, but autoprefixer uses 7.0.2. Perhaps this is the source of the error below.
```

**Comment by ai** [comment]
I think it is just the error. Open an issue and post error message with a full stack trace and input CSS.

**Comment by redaxmedia** [comment]
Andrey, I hope it is okay that you clone into my project and give it a try?

Master is working: https://travis-ci.org/redaxscript/redaxscript/builds/429001005
Debug branch is broken: https://travis-ci.org/redaxscript/redaxscript/builds/429001658

1. Please install the project: https://github.com/redaxscript/redaxscript#installation
2. Run `grunt build` and you will see it is working on the master
3. Update to latest `postcss-custom-media` and run `grunt build` again
4. You can find the PostCSS setup in [tasks/postcss.js](https://github.com/redaxscript/redaxscript/blob/master/tasks/postcss.js#L193-L216)

Thank you, I hope it this workflow is okay for you...

**Comment by jonathantneal** [comment]
Thanks for reporting this error. I believe this is resolved in v7.0.2 https://github.com/postcss/postcss-custom-media/commit/407c33cd8821a869db9dc6fac3d99681ee74a1ef.

**Comment by redaxmedia** [comment]
Thank you, seems to be working now!
